package.json

```diff
@@ -27,17 +27,17 @@
     "docs:build": "cd ./website && npm run build && cd ..",
     "docs:deploy": "./scripts/updateDocs.js",
     "test": "run-s test:*",
-    "test:eslint": "eslint packages examples scripts tests",
+    "test:eslint": "eslint --cache packages examples scripts tests",
     "test:typings": "npm run generate:typings && node tests/typings/copy && npm run ts && npm run clean:tests",
-    "test:coverage": "jest --coverage --detectOpenHandles",
+    "test:coverage": "jest --coverage",
     "test:smoke": "babel-node ./tests/smoke.runner.js",
     "ts": "run-p ts:*",
-    "ts:sync": "cd tests/typings/sync && tsc",
-    "ts:sync-mocha": "cd tests/typings/sync-mocha && tsc",
-    "ts:sync-jasmine": "cd tests/typings/sync-jasmine && tsc",
-    "ts:webdriverio": "cd tests/typings/webdriverio && tsc",
-    "ts:webdriverio-mocha": "cd tests/typings/webdriverio-mocha && tsc",
-    "ts:webdriverio-jasmine": "cd tests/typings/webdriverio-jasmine && tsc",
+    "ts:sync": "cd tests/typings/sync && tsc --incremental",
```

Comment:
What does `--incremental` doing?

Reply:
> Enable incremental compilation by reading/writing information from prior compilations to a file on disk.

> TypeScript 3.4 introduces a new flag called --incremental which tells TypeScript to save information about the project graph from the last compilation. The next time TypeScript is invoked with --incremental, it will use that information to detect the least costly way to type-check and emit changes to your project

Reply:
it generates `tsconfig.tsbuildinfo` file in /tests/types/**/dist folder